_inc/client/components/jetpack-logo/index.jsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React from 'react';
+
+class JetpackLogo extends React.Component {
+	static propTypes = {
+		className: PropTypes.string,
+		width: PropTypes.number,
+		height: PropTypes.number,
+		x: PropTypes.string,
```

Comment:
I think `x`, `y` and `viewBox` are kind of unnecessary here - I doubt anyone would want to pass them as props. Let's remove them and hardcode them in the svg?

Reply:
This has been addressed in https://github.com/Automattic/jetpack/pull/13348/commits/ba63eeeffc7502a1663067a0c7a74b5f18496975.